extensions/sherlock-proxy/proxy.ts

```diff
@@ -94,7 +94,7 @@ export class ProxyDescriptor<V = any, T = V> {
         const pd = this.$proxyDescriptor;
         try {
             return pd.$derivable.get();
-        } catch (e) {
+        } catch (e: any) {
```

Comment:
`unknown` is safer than `any` because it reminds us that we need to perform some sorts of type-checks before operating on our values.
https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#unknown-on-catch

Reply:
But if it is an `unknown`, we cannot do `e.message` on line 99

Reply:
Looking good (y)